src/components/Common/Hamburger/Hamburger.tsx

```diff
@@ -7,13 +7,12 @@ import {
   type MenuProps,
   type MenuTriggerProps,
 } from 'react-aria-components'
+import styles from './Hamburger.module.scss'
```

Comment:
Super nit: i like my relative imports last 😂 curious what other preferences there are on the team. Definitely definitely not a blocker

Reply:
Feel free to change it! I just want some order 😆